src/prometheus_fastapi_instrumentator/metrics.py

```diff
@@ -27,6 +27,7 @@ def __init__(
         modified_handler: str,
         modified_status: str,
         modified_duration: float,
+        custom_labels: dict,
```

Comment:
I prefer to not have this attribute in the `Info` class. It should only contain attributes that are relevant for the respective request / response pair.

It is sufficient if all metrics closures included with this package get it as an additional parameter.